doc/toolbox-create.1.md

```diff
@@ -17,19 +17,21 @@ to interact with the container at any point.
 A toolbox container is an OCI container created from an OCI image. On Fedora,
 the default image is known as `fedora-toolbox:N`, where N is the release of
 the host. If the image is not present locally, then it is pulled from a
-well-known registry like `registry.fedoraproject.org`. The container is
-created with `podman create`, and its entry point is set to `toolbox
-init-container`.
+well-known registry like `registry.fedoraproject.org`. Other images may be
+used on other host operating systems. If the host is not recognized, then the
+Fedora image will be used.
 
-By default, toolbox containers are named after their corresponding images. If
-the image had a tag, then the tag is included in the name of the container,
-but it's separated by a hyphen, not a colon. A different name can be assigned
-by using the CONTAINER argument.
+The container is created with `podman create`, and its entry point is set to
+`toolbox init-container`.
 
-If the host system is not recognized the used image will be `fedora-toolbox`.
+By default, a toolbox container is named after its corresponding image. If the
+image had a tag, then the tag is included in the name of the container, but
+it's separated by a hyphen, not a colon. A different name can be assigned by
+using the CONTAINER argument.
 
-Toolbox containers are primarily created in a way to be tightly integrated with
```

Comment:
I wouldn't drop completely the mention of toolbox containers not being secure.

Reply:
First, if we really want to talk about security, then it's not nice to slip in a sentence like *not meant to be secure* in the middle of a second-level manual like `toolbox-create(1)`. It should really be mentioned a lot more clearly in somewhere more visible. Like `README.md` or `toolbox(1)`.

Second, words like *privileged* and *secure* are very loaded and have no meaning, unless they are clearly explained at some length. Casually mentioning those words in passing do no good, because everybody interprets them differently.

What's the definition of secure? How is it insecure? How is it insecure than any other software running on the system? Is Bash secure - it lets me run `rm -rf *`? Why are we even creating something that's insecure and hence harmful to our users?

We clearly mention what Toolbox does, the context in which it exists, and people can draw their own conclusions if that fits their definition of *secure*.

We can definitely have a section about what role Toolbox plays alongside things like OSTree-based OSes and Flatpak, because only then we can say something meaningful about security. However, that should be somewhere top-level like `README.md` or `toolbox(1)`, not `toolbox-create(1)`.